src/System.Security.Cryptography.Xml/src/Configurations.props

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard
```

Comment:
We agreed to have a netstandard ref and src as well as a netfx src which will be a partial facade.